testing/rabbitmq/APKBUILD

```diff
@@ -0,0 +1,91 @@
+# Contributor: Nathan Johnson <nathan@nathanjohnson.info>
+# Maintainer: Nathan Johnson <nathan@nathanjohnson.info>
+pkgname=rabbitmq
+_realname=rabbitmq-server
```

Comment:
Why you’ve named it `rabbitmq` instead of `rabbitmq-server`, that seems to be the official name?